electrum/gui/qt/amountedit.py

```diff
@@ -45,6 +45,9 @@ def numbify(self):
         if text == '!':
             self.shortcut.emit()
             return
+        if ',' in text:
```

Comment:
imho to be sure honest with all users we should NEVER
change the value that was provided
let's validate it and show to the user that he cannot:
- use `,` at all
- use multiple `.`

leading zeros can be stripped but only if:
- `.` in the value
- `value[0] != 0`

Reply:
done